_src/sitemap.xml

```diff
@@ -0,0 +1,444 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>https://www.bitdefender.com.au/solutions/internet-security</loc>
```

Comment:
after updating create.js, this will look like <loc>https://www.bitdefender.com/zh-hk/solutions/internet-security</loc>